### DISCUSSION TITLE
PWA and App Store

### DISCUSSION THREAD

**Discussion by luke-cf** [ionic-team/capacitor#3579] [Q&A]
I would like to use a video player written in javascript. Can it be done in PWA, bearing in mind that the application is to be made available in the app store?

**Comment by jcesarmobile** [comment]
A PWA is a web you can "install" from the browser, not from the App Store.
But with Capacitor you can use the same code and create a hybrid app that you can publish on the App Store.

**Reply by luke-cf** [reply]
Thanks for the answer. Yes, I know what PWA is and how it works. I just wonder if a project written in this way, for example in Angular, can be published in an App store. I heard there may be acceptance issues. Do you have experience publishing such an application? Will be grateful for a description of your experience with this.

Am I to understand that in my case Capacitor is a wrapper for PWA and I can use e.g. a video player written in javascript?

**Reply by jcesarmobile** [reply]
If you point to the server url where you have the PWA running, then yeah, there could be acceptance issues.
If you package the app assets inside the app then there are no acceptance issues.

Capacitor is not a wrapper for PWAs, but lets you use the same code you use in your PWA.
